<launch>
    <group ns="hardware">
        <rosparam command="load" file="$(find hardware)/params/robot_id.yaml" />
        <rosparam command="load" file="$(find hardware)/params/pid.yaml" />
        <!-- Node to connect with micro-controller -->
        <node name="robot_controller" pkg="rosserial_python" type="serial_node.py" required="true" output="screen" >
            <param name="port" value="/dev/ttyUSB0"/>
            <param name="baud" value="200000"/>
        </node>
        <!-- Node to get data from hardware -->
        <node name="robot_introspection_node" pkg="hardware" type="robot_introspection_node.py" />
        <node name="light_sensors_node" pkg="hardware" type="light_sensors_node" />
        <node name="laser_scan_node" pkg="hardware" type="laser_scan_node" output="screen" />
    </group>
</launch>
